src/core/components/ChatHistory.module.css

```diff
@@ -78,6 +78,7 @@
   flex: 1;
   min-width: 0;
   cursor: pointer;
+  display: flex;
```

Comment:
In order for the `flex-grow: 1` on the input to take effect, its parent element also need `display: flex`